pyveda/models/main.py

```diff
@@ -123,13 +129,15 @@ def predict(self, bounds, image=None, **kwargs):
         meta = {
             "name": kwargs.get('name', self.name),
             "description": kwargs.get("description", None),
-            "dtype": self.dtype, 
+            "dtype": self.dtype,
             "mltype": self.mltype,
             "imshape": kwargs.get('imshape', self.imshape),
             "public": False,
             "bounds": bounds,
             "classes": self.classes,
             "channels_last": str(self.channels_last)
+            # "weights_file_name" = self.weights_file_name,
+            # "model_file_name" = self.model_file_name
```

Comment:
These should be removed. The idea is that since the model object has them, the predictionsets dont really need them. They can be found by fetching the model from vector services.

Reply:
removed